geniza/corpus/management/commands/export_metadata.py

```diff
@@ -0,0 +1,28 @@
+import time
+from asyncore import write
```

Comment:
what is this? (do you by any chance have the vs code setting turned on that tries to autoimport for you? that was always adding weird imports to my code until I found it and turned it off)